client/client-lib/src/api.rs

```diff
@@ -125,7 +125,7 @@ impl<'a> dyn FederationApi + 'a {
 #[derive(Debug)]
 pub struct WsFederationApi<C = WsClient> {
     members: Vec<FederationMember<C>>,
-    max_evil: usize,
+    pub peers: Vec<PeerId>,
```

Comment:
In how far could we reuse the `members` field here? Maybe its type could become `BTreeMap<PeerId, FederationMember<C>>`?

Reply:
`FederationMember` already contains `PeerId`, so I can expose it as a function.